lib/funky/page.rb

```diff
@@ -0,0 +1,43 @@
+module Funky
+  class Page < FacebookGraphRootNode
+    # For further detailed documentation, see
+    # https://developers.facebook.com/docs/graph-api/reference/page/
```

Comment:
@bruce-fs According to Yardoc you should move this comment *before* the class, and use the `@see` keyword, so

```ruby
# @see https://developers.facebook.com/docs/graph-api/reference/page/
class Page < ...
```